frontend/storage/models/Game.ts

```diff
@@ -0,0 +1,14 @@
+export interface GameProps{
+	gameId: number;
+	gameName: string;
+}
+export class Game {
+	public readonly gameId: number;
+
+	public readonly gameName: string;
```

Comment:
`game` in the name of this variable is redundant, whenever you interact with this you already know  that you're talking about the name of the game.

Ditto for `gameId`, just change both to `id` and `name` :).

On the other hand there is an argument for "namespacing" the ID with the name of the class. In MySQL there is a short form join , where you can do `JOIN <x> USING (game_id)` instead of `JOIN <x> ON (left.id = right.game_id)` but that's a marginal benefit. And it also tempts you to not have meaningful column names.